yt_dlp/extractor/tbsjp.py

```diff
@@ -0,0 +1,151 @@
+from .common import InfoExtractor
+from ..networking.exceptions import HTTPError
+from ..utils import (
+    ExtractorError,
+    clean_html,
+    get_element_text_and_html_by_tag,
+    int_or_none,
+    str_or_none,
+    traverse_obj,
+    try_call,
+    unified_timestamp,
+    urljoin,
+)
+
+
+class TBSJPEpisodeIE(InfoExtractor):
+    _VALID_URL = r'https://cu\.tbs\.co\.jp/episode/(?P<id>[\d_]+)'
+    _TESTS = [{
+        'url': 'https://cu.tbs.co.jp/episode/23613_2044134_1000049010',
+        'skip': 'streams geo-restricted, Japan only. Also, will likely expire eventually',
+        'info_dict': {
+            'title': 'VIVANT 第三話 誤送金完結へ!絶体絶命の反撃開始',
+            'id': '23613_2044134_1000049010',
+            'ext': 'mp4',
+            'upload_date': '20230728',
+            'duration': 3517,
+            'release_timestamp': 1691118230,
+            'episode': '第三話 誤送金完結へ!絶体絶命の反撃開始',
+            'release_date': '20230804',
+            'categories': 'count:11',
+            'episode_number': 3,
+            'timestamp': 1690522538,
+            'description': 'md5:2b796341af1ef772034133174ba4a895',
+            'series': 'VIVANT',
+        },
+    }]
+
```

Comment:
if geo bypass doesn't work then we should do this so that `raise_geo_restricted` doesn't cause a retry w/xff
```suggestion
    _GEO_BYPASS = False

```